test/2_create_test_app_namespace.sh

```diff
@@ -18,9 +18,9 @@ else
   echo "Creating '$TEST_APP_NAMESPACE_NAME' namespace."
 
   if [ $PLATFORM = "kubernetes" ]; then
-    $cli create namespace $TEST_APP_NAMESPACE_NAME
+    $cli "create namespace $TEST_APP_NAMESPACE_NAME"
```

Comment:
what's the difference here?

Reply:
If it's important to add quotes please say so in the commit and split all these refactors into one commit.

Reply:
This is almost all the changes you will see in the files (this is why i didnt split the PR). since i added to cli the retry timeout i have to encapsulate the command i'm working with.

Reply:
Still I believe it should be separated, and the commit message can explain why this is needed. Also - it will make the review much easier as the reviewer will understand the commit, and just skip it instead of going over all of these lines looking for a change other than adding quotes.

Reply:
i will do this for the next time because each change i'm adding i'm verifying 5 times to make sure no issues. and since almost all the files contain 1 change i guess it will be o.k. 
Thanks for this comment.

Reply:
ok. However, if you're already making changes you'll need to push the commits so changing the original commits will not start any additional build.